src/otx/recipe/detection/rtdetr_50.yaml

```diff
@@ -53,10 +53,11 @@ overrides:
       transforms:
         - class_path: torchvision.transforms.v2.RandomPhotometricDistort
           init_args:
-            p: 0.5
+            p: 0.0
```

Comment:
Why did you switch off the RandomPhotometricDistort and RandomZoomOut?

Reply:
I think it can hurt the generalization of the model. We don't have augmentations now except for Flip.

Reply:
@kprokofi The accuracy is very bad (almost 0%) on several dataset I benchmarked. This is due to strong augmentation applied.

Reply:
I think we can keep RandomPhotometricDistort, but RandomZoomOut might over-augment the image.

Reply:
@eugene123tw did you run a quick training on a medium to check is the new setting are ok?

Reply:
I agree, we can remove RandomZoomOut, but RandomPhotometricDistort should not influence so much, it is very common augmentation, almost basic one. We can reduce probability though. Also, if you want to switch off the augmentation, could you remove it at all from data recipe? I don't think it makes sense to keep it in recipe with probability = 0

Reply:
@sovrasov I ran it on pothole_medium
test/f1-score with both aug = 0.5822
test/f1-score disable both   =   0.4100
test/f1-score with RandomPhotometricDistort = 0.6271